pkg/distro/defs/fedora/distro.yaml

```diff
@@ -133,6 +178,37 @@
           - "grub2-ppc64le"
           - "grub2-ppc64le-modules"
       bootloader: "grub2"
+    ppc64le_installer_platform: &ppc64le_installer_platform
+      <<: *ppc64le_bios_platform
+      image_format: "qcow2"
+      packages:
+        <<: *ppc64le_bios_platform_packages
+        booting:
+
```

Comment:
(super nitpick) is the extra newline here on purpose?

Reply:
Nope :)